scout/adapter/mongo/case.py

```diff
@@ -775,6 +782,7 @@ def update_case(self, case_obj, keep_date=False):
                     "mme_submission": case_obj.get("mme_submission"),
                     "status": case_obj.get("status"),
                     "variants_stats": case_obj.get("variants_stats"),
+                    "track": case_obj.get("track"),
```

Comment:
track was not updated on re-upload!

Reply:
😅  There was this concept in computer science called OOD. It could have been useful here when propagating info  for this object here. A bit of inheritance with polymorphism could have made this a little easier. Anyway, this is in the adapter, so all good I suppose. Was that the only missing one?

Reply:
Maybe we should at the very least sort according to either alphabet or the order in the "models" object. Pydantic may help a bit as well later.

Reply:
I can sort alphabetically, and sort the model as well. I'll check if we forgot anything with the model object

Reply:
Did the alphabetical sorting and found out that using models for checking that all the keys are taken into account in adapters is actually not the best way to act --> In fact I've found 2 keys that are used in case adapter but not found in case model (i've added them now).

I think a more comprehensive check of all the possible keys of a case (some might still be missing in the model) and their use in this adapter function is beyond the purpose of this fix. Hopefully using pydantic will help find more of these issues!